### ISSUE TITLE
Vuestic simple and multiselect problems

### ISSUE THREAD

**Issue by pfrohner** [epicmaxco/vuestic-admin#186]
I'm trying to make these work in the project I've been working on and I ran into the following issues:

 - With the simple select, there is no way to un-select a value, would be nice to have an option to do so, since a field can be optional.
 - Tried to use multiselect instead, because at least you can un-select things there. Unfortunately there is no way to load the previously stored selections into the component via data or any other way (works with simple select).

**Comment by smartapant** [comment]
@pfrohner thanks for mentioning, will be having a look into the issue

**Comment by vlad-shusterman** [comment]
1)already implemented.
2)Could you tell more information on this point(maybe show the case, where we can use this feature)?

**Comment by pfrohner** [comment]
@vlad-shusterman
1) Where can I see that? I just pulled the repo, it doesn't work. I can click on an item twice but then I got an [object Object] as the value....
2) http://vuestic.epicmax.co/#/forms/form-elements there is the multiselect, let's say we have previously stored some values selected by the user and we'd like to display them on page load. Usually, that goes by setting the v-model to that stored value, and passing it to the component. How can I achieve that?
